#! /usr/bin/env python
# Task Long configuration

PACKAGE='sim_tasks'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("angle",         double_t, 0,    "Goal angle between the boat and the shore",  3.14)
gen.add("rotation_vel",  double_t, 0,    "Rotation absolute velocity of the boat",  0.5)
gen.add("angle_range",   double_t, 0,    "Range on each side of the reference angle, where we detect the points",  0.78)

exit(gen.generate(PACKAGE, "sim_tasks", "TaskRotate"))

